core/trino-server-core/src/main/provisio/trino.xml

```diff
@@ -0,0 +1,96 @@
+<runtime>
+    <!-- Target -->
+    <archive name="${project.artifactId}-${project.version}.tar.gz" hardLinkIncludes="**/*.jar" />
+
+    <!-- Notices -->
+    <fileSet to="/">
+        <directory path="${basedir}">
+            <include>NOTICE</include>
+            <include>README.txt</include>
+        </directory>
+    </fileSet>
+
+    <!-- Launcher -->
+    <artifactSet to="bin">
+        <artifact id="io.airlift:launcher:tar.gz:bin:${dep.launcher.version}">
+            <unpack />
+        </artifact>
+        <artifact id="io.airlift:launcher:tar.gz:properties:${dep.launcher.version}">
+            <unpack filter="true" />
+        </artifact>
+    </artifactSet>
+
+    <!-- Server -->
+    <artifactSet to="lib">
+        <artifact id="${project.groupId}:trino-server-main:${project.version}" />
```

Comment:
We should modify the full server to consume this core package. Otherwise, we'd have to duplicate some product test to test this one, at least the [SuiteAllConnectorsSmoke] to make sure all plugins are placed here correctly.

Reusing this as a dependency will also mean new plugins only need to be added to one package - this is an easy mistake to do.

Reply:
Yeah ... I wasnt sure if thats worth the effort or not.. let me play with it some more.

Reply:
Reuse is working now.